src/ch/ch_conf.c

```diff
@@ -171,6 +171,62 @@ virCHDriverConfigDispose(void *obj)
     g_free(cfg->logDir);
 }
 
+int
+virCHVersionString(const char *str, unsigned long *version,
+                      bool allowMissing)
```

Comment:
`allowMissing` is unsed? what is its purpose?

Reply:
@praveen-pk Remove. Please review latest push